gitops/addons/charts/adot-collector/values.yaml

```
@@ -0,0 +1,41 @@
+ampurl: ""
+region: ""
+ekscluster: "" 
+accountId: ""
+
+assumeRoleArn: ""
+
+globalScrapeTimeout: 15s
+globalScrapeInterval: 60s
+globalScrapeSampleLimit: 60s
+
+enableAPIserver: true
+
+enableTracing: false
+otlpGrpcEndpoint: ""
+otlpHttpEndpoint: ""
+tracingTimeout: ""
+tracingSendBatchSize: ""
```

```suggestion
tracingBatchSize: ""
```
change variable name to match the one in template